CHANGES.md

```diff
@@ -61,6 +61,7 @@
 
 ## I/Os
 
+* Added support for handling bad records to BigQueryIO ([#30081](https://github.com/apache/beam/pull/30081)).
```

Comment:
Can we mention the specific read/write modes?

Reply:
good idea, added